integrations/ultralytics-yolov3/README.md

```diff
@@ -14,3 +14,112 @@ See the License for the specific language governing permissions and
+
+## Tutorials
+
+- Coming soon!
+
+## Installation
+
+To begin using this integration, run the following command in the root directory:
+```bash
+bash setup_integration.sh
+```
+
+The setup_integration.sh file will clone the yolov3 repository with the SparseML integration as a subfolder.
+After the repo has successfully cloned,  all dependencies from the `yolov3/requirements.txt` file will install in your current environment.
+
+## Quick Tour
+
+### Recipes
+
+Recipes encode the instructions and hyperparameters for sparsifying a model using modifiers to the training process.
+The modifiers can range from pruning and quantization to learning rate and weight decay.
+When appropriately combined, it becomes possible to create highly sparse and accurate models.
+
+This integration adds a `--recipe` argument to the [`train.py` script](https://github.com/neuralmagic/yolov3/blob/master/train.py).
+The argument loads an appropriate recipe while preserving the rest of the training pipeline.
+Popular recipes used with this argument are found in the [`recipes` folder](./recipes).
+Otherwise, all other arguments and functionality remain the same as the original repository.
+
+For example, pruning and quantizing a model on the VOC dataset can be done by running the following command from within the yolov3 repository folder:
+```bash
+python train.py --batch 16 --img 512 --weights PRETRAINED_WEIGHTS \
+    --data voc.yaml --hyp ../data/hyp.pruned_quantized.yaml \
+    --recipe ../recipes/yolov3-spp.pruned_quantized.short.md
+```
+
+### SparseZoo
+
+Pre-sparsified models and recipes can be downloaded through the [SparseZoo](https://github.com/neuralmagic/sparsezoo).
+The following popular stubs can be used to download these models and recipes through the SparseZoo API directly.
+
+| Model                       | Dataset | mAP@50 | Description                                                    | SparseZoo Stub                                                                   |
+|-----------------------------|---------|--------|----------------------------------------------------------------|----------------------------------------------------------------------------------|
+| YOLOv3-SPP Pruned Quantized | COCO    | 60.4   | 83% sparse and INT8 quantized model with LeakyReLU activations | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned_quant-aggressive_94 |
+| YOLOv3-SPP Pruned           | COCO    | 62.4   | 88% sparse model with LeakyReLU activations                    | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/pruned-aggressive_97       |
+| YOLOv3-SPP                  | COCO    | 64.2   | Baseline model with LeakyReLU activations                      | zoo:cv/detection/yolo_v3-spp/pytorch/ultralytics/coco/base-none                  |
+
+Web tables are available for complete lists for all [models](https://sparsezoo.neuralmagic.com/tables/models/cv/detection?repo=ultralytics) and 
```

Comment:
What are "web tables" @markurtz or @bfineran?
Is it simply "Complete lists are available online for alll..."

Reply:
yes, updated to clarify with that suggestion